lib/compact_index/versions_file.rb

```diff
@@ -15,80 +15,34 @@ def contents(gems = nil, args = {})
 
       raise ArgumentError, "Unknown options: #{args.keys.join(", ")}" unless args.empty?
 
-      out = File.read(@path)
-      out << parse_gems(gems) if gems
-      out
+      File.read(@path).tap do |out|
+        out << gem_lines(gems) if gems
+      end
     end
 
     def updated_at
       created_at_header(@path) || Time.at(0).to_datetime
     end
 
-    def update_with(gems)
```

Comment:
`update_with` was used in [the rake task](https://github.com/bundler/bundler-api/blob/299e94ba72edb2825952b361b0101a41d16a08fe/Rakefile#L211) which updated versions file. Does this mean we will always `create` the file?